scripts/start_arm64_local_marqo_os.sh

```diff
@@ -27,6 +27,8 @@ set -x
 docker run -d --name marqo --privileged -p 8882:8882 --add-host host.docker.internal:host-gateway \
     -e "MARQO_MAX_CPU_MODEL_MEMORY=1.6" \
     -e "OPENSEARCH_URL=$LOCAL_OPENSEARCH_URL" \
+    -e "MARQO_MAX_BACKEND_SEARCH_RETRY_ATTEMPTS='5'" \
```

Comment:
We usually don't hardcode env vars for our tests directly into our scripts like this. Instead this could be a better approach:

1. Remove `MARQO_MAX_BACKEND_SEARCH_RETRY_ATTEMPTS` and `MARQO_MAX_BACKEND_ADD_DOCS_RETRY_ATTEMPTS` from scripts.
2. Move the tests under the `application_tests` dir, as it will involve stopping/restarting marqo.
3. Put the tests under `test_env_var_changes`: https://github.com/marqo-ai/marqo-api-tests/blob/mainline/tests/application_tests/test_env_var_changes.py#L28
4. At the beginning of the test, run `utilities.rerun_marqo_with_env_vars()` to set the env vars to what you want

This is more flexible, and it means you can test more values for `MARQO_MAX_BACKEND_SEARCH_RETRY_ATTEMPTS` and `MARQO_MAX_BACKEND_ADD_DOCS_RETRY_ATTEMPTS`.

Reply:
Great suggestion! thanks for this!